ledger/src/blockstore_db.rs

```diff
@@ -984,6 +992,14 @@ impl TypedColumn for columns::ErasureMeta {
     type Type = blockstore_meta::ErasureMeta;
 }
 
+impl SlotColumn for columns::OptimisticSlot {}
+impl ColumnName for columns::OptimisticSlot {
+    const NAME: &'static str = OPTIMISTIC_SLOT_CF;
```

Comment:
OPTIMISTIC_SLOTS_CF

Reply:
There's a mix of singular and plural in the _cf naming. I'd prefer to leave this as-is.